packages/legend-server-sdlc/src/models/configuration/ProjectDependency.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import { createModelSchema, primitive } from 'serializr';
+import { createModelSchema, primitive, alias } from 'serializr';
```

Comment:
Why can't we just change `versionId: VersionId` to `version: string`? What does it cost? The reason why I'm not 100% through with this change is that I don't see @kevin-m-knight-gs making the same sort of change in https://github.com/finos/legend-sdlc/pull/442

Reply:
The reason I didn't change the `versionId: VersionId ` to `version: string` is because of the external packages which might be reading the `versionId` class from the project Dependency.

If I change the versionId to a string then external projects would also need to make the same changes when reading it from the legend-studio

And the new sdlc payload structure require a string in place of the class object value.

Reply:
That's fair point, but I think we can prioritize keeping this consistent with sdlc model. We will release this change in a major version

Reply:
Ok.. So should I change the  the `versionId: VersionId` to `versionId: string` so the we'll be consistent with the sdlc model ?
Or leave it as I've implemented ?

Reply:
let's do `versionId: string`